### ISSUE TITLE
Regex capture groups are not supported in Deck yaml.

### ISSUE THREAD

**Issue by DanielRailean** [Kong/deck#672]
### TLDR 
We followed this [Kong support article](https://support.konghq.com/support/s/article/How-to-proxy-request-to-a-specific-upstream-path-when-the-upstream-root-path-already-exists) to strip part of the path sent to the upstream.

The problem is that while you can do that in Kong , you cannot do that with deck.

### Route in Kong : 

![165692588-c1d01a9e-a177-4a2d-800a-3bf23f99aeb9 (1)](https://user-images.githubusercontent.com/29011940/166884163-462ffaca-a43b-48d4-90eb-6d478f6655ef.png)

### API Spec we are sending : 

![MicrosoftTeams-image (1) (1)](https://user-images.githubusercontent.com/29011940/166883583-ba76b058-c52e-4db0-81cf-97813fc81d51.png)

## Deck error.

![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/29011940/166884344-dcb9be47-d3d6-481e-922a-e4d5800d5e65.png)

**Comment by GGabriele** [comment]
Closing this, please feel free to comment back if you have any question!
